packages/core/src/components/Badge/Badge.tsx

```diff
@@ -20,15 +20,15 @@ export interface BadgeBaseProps extends VibeComponentProps {
   children: React.ReactNode;
 }
 
-interface CounterBadgeProps extends CounterProps {
+export interface CounterBadgeProps extends CounterProps {
   type: BadgeType.COUNTER;
 }
 
-interface IndicatorBadgeProps extends IndicatorProps {
+export interface IndicatorBadgeProps extends IndicatorProps {
```

Comment:
I believe those can be considered as internal props for the component, are you sure they need to be exported?

Reply:
Removed thanks!